common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -1053,3 +1057,27 @@ def test_team_full_message(self):
         self.team_page.visit()
         self.assertEqual(self.team_page.join_team_message, 'This team is full.')
         self.assert_team_details(num_members=1, is_member=False, max_size=1)
+
+    def test_leave_team(self):
+        """
+        Scenario: User can leave a team.
+
+        Given I am enrolled in a course with a team configuration, a topic,
+            and a team belonging to that topic
+        And I am a member of team
+        And I visit the team
+        And I should not see Join Team button
+        And I should see New Post button
+        Then I should see Leave Team link
+        When I click on Leave Team link
+        Then user should be left from team
+        And I should see Join Team button
+        And I should not see New Post button
+        """
+        self._set_team_configuration_and_membership()
+        self.team_page.visit()
+        self.assertFalse(self.team_page.join_team_button_present)
+        self.assert_team_details(num_members=1, invite_text='Send this link to friends so that they can join too.')
```

Comment:
Setting `is_member=True` explicitly here will make this clearer. Also the invite string should be made into a constant.